tests/scripts/remote_monitoring_tests/remote_monitoring_tests.sh

```diff
@@ -37,13 +37,7 @@ function remote_monitoring_tests() {
 	((TOTAL_TEST_SUITES++))
 
 	python3 --version >/dev/null 2>/dev/null
-	err=$?
-	if [ ${err} -ne 0 ]; then
-		echo "ERROR: python3 not installed"
-		exit 1
-	fi
-	
-	python3 -m pip install --user -r "${REMOTE_MONITORING_TEST_DIR}/requirements.txt" >/dev/null 2>&1
```

Comment:
Is it intentional to remove this install package line as well?

Reply:
I have moved it below.